autoware_launch/config/perception/object_recognition/detection/lidar_model/centerpoint.param.yaml

```diff
@@ -13,3 +13,15 @@
     iou_nms_search_distance_2d: 10.0
     iou_nms_threshold: 0.1
     yaw_norm_thresholds: [0.3, 0.3, 0.3, 0.3, 0.0]
+    score_threshold: 0.35
+    build_only: false # shutdown node after TensorRT engine file is built
```

Comment:
Was not this deleted in
https://github.com/autowarefoundation/autoware.universe/pull/6167 ?
:bow:

Reply:
Thanks, fixed: https://github.com/autowarefoundation/autoware_launch/pull/822/commits/d762ab1c7babc14511d30c9ce8e909a7ad6332ba